File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    var min = Math.min(n, m);
    var num = 0;
    for(var i = 1; i <= min; i++) {
        if(n % i === 0 && m % i === 0) num = i;
    }
    return [num, n * m / num];
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int min = Math.min(n, m);
        int ans = 1;
        for(int i = 1; i <= min; i++) {
            if(n % i == 0 && m % i == 0)
                ans = i;
        }
        return new int[]{ans, n * m / ans};
    }
}